<ui version="4.0" >
 <class>Client</class>
 <widget class="QWidget" name="Client" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Personal Info</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Birth</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="9" column="0" >
        <widget class="QLabel" name="label_18" >
         <property name="text" >
          <string>Zone</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2" >
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>Latitude</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QComboBox" name="latitude_combo" />
         </item>
         <item>
          <widget class="QLineEdit" name="latitude_deg_line" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask" >
            <string>999; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_27" >
           <property name="text" >
            <string>deg</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="latitude_min_line" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask" >
            <string>99; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_28" >
           <property name="text" >
            <string>min</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="2" colspan="2" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QComboBox" name="longitude_combo" />
         </item>
         <item>
          <widget class="QLineEdit" name="longitude_deg_line" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask" >
            <string>999; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_25" >
           <property name="text" >
            <string>deg</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="longitude_min_line" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask" >
            <string>99; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_26" >
           <property name="text" >
            <string>min</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Longitude</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="city_line" />
         </item>
         <item>
          <widget class="QPushButton" name="find_city_button" >
           <property name="text" >
            <string>Find City</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>City</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Sex</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QComboBox" name="sex_combo" />
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="3" >
        <widget class="QLineEdit" name="maiden_line" />
       </item>
       <item row="1" column="0" colspan="3" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Maiden Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2" >
        <widget class="QLineEdit" name="name_line" >
         <property name="inputMask" >
          <string/>
         </property>
         <property name="echoMode" >
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Birth Date</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="2" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="hour_line" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask" >
            <string>99; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_22" >
           <property name="text" >
            <string>hour</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="minute_line" >
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask" >
            <string>99; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_23" >
           <property name="text" >
            <string>min</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="7" column="0" colspan="2" >
        <widget class="QLabel" name="label_17" >
         <property name="text" >
          <string>Birth Time</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="year_line" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask" >
            <string>0000; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>year</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="month_line" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask" >
            <string>99; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>month</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="day_line" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask" >
            <string>99; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_21" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>day</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="birthdate_button" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="8" column="0" colspan="4" >
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="3" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="set_zone_button" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>1</vsizetype>
             <horstretch>254</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Set >></string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="zone_combo" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>4</vsizetype>
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="dst_check" >
           <property name="text" >
            <string>DST</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Contact</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="phone1_line" />
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="phone2_line" />
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="mobile1_line" />
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_12" >
           <property name="text" >
            <string>Phone1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="mobile2_line" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_13" >
           <property name="text" >
            <string>Mobile1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_15" >
           <property name="text" >
            <string>Mobile2</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QLineEdit" name="email_line" />
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>e-mail</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_14" >
           <property name="text" >
            <string>Phone2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="address_text" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Life events</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QTableWidget" name="events_table" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Comments</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QTextEdit" name="comment_text" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="apply_button" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>name_line</tabstop>
  <tabstop>maiden_line</tabstop>
  <tabstop>sex_combo</tabstop>
  <tabstop>longitude_deg_line</tabstop>
  <tabstop>longitude_min_line</tabstop>
  <tabstop>latitude_deg_line</tabstop>
  <tabstop>latitude_min_line</tabstop>
  <tabstop>zone_combo</tabstop>
  <tabstop>year_line</tabstop>
  <tabstop>month_line</tabstop>
  <tabstop>day_line</tabstop>
  <tabstop>birthdate_button</tabstop>
  <tabstop>hour_line</tabstop>
  <tabstop>minute_line</tabstop>
  <tabstop>dst_check</tabstop>
  <tabstop>phone1_line</tabstop>
  <tabstop>phone2_line</tabstop>
  <tabstop>mobile1_line</tabstop>
  <tabstop>mobile2_line</tabstop>
  <tabstop>email_line</tabstop>
  <tabstop>address_text</tabstop>
  <tabstop>comment_text</tabstop>
  <tabstop>apply_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
